src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/TargetPlatformAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+    /// <summary>
+    /// Records the platform that the project targeted.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly,
+                    AllowMultiple = false, Inherited = false)]
+    public sealed class TargetPlatformAttribute : PlatformAttribute
+    {
+        public TargetPlatformAttribute(string platformName) : base(platformName)
+        {
+        }
+    }
+}
```

Comment:
Please add tests for all of the attributes, ala https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/tests/System/Diagnostics/CodeAnalysis/DynamicDependencyAttributeTests.cs

Reply:
@jeffhandley do you have someone available to write these tests? As I stated before, I don't know how to build runtime repo. And I am also having my own deliverables close to deadline.

Reply:
Yeah, we can get the unit tests added from the .NET Libraries team side.  I'd prefer to have that be a PR into your fork/branch so that can then be added into this PR and it all gets merged together.  I'll get back to you in a few hours if we'll use that approach or a different approach.

Reply:
I can add the tests